ts/repositories.bzl

```diff
@@ -6,24 +6,85 @@ See https://docs.bazel.build/versions/main/skylark/deploying.html#dependencies
 
 load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
 load("@bazel_tools//tools/build_defs/repo:utils.bzl", "maybe")
+load("//ts/private:versions.bzl", TS_VERSIONS = "VERSIONS")
 
 versions = struct(
     bazel_lib = "0.9.7",
     rules_nodejs = "5.4.0",
     rules_js = "1e97acf805b11d7b8716a5c2f4d4466f57d3fb3d",
-    # Users can easily override the typescript version just by declaring their own http_archive
-    # named "npm_typescript" prior to calling rules_js_dependencies.
-    typescript = "4.6.3",
 )
 
-# WARNING: any changes in this function may be BREAKING CHANGES for users
+LATEST_TS_VERSION = TS_VERSIONS.keys()[-1]
```

Comment:
I thought order in maps was not guaranteed? Is that not true for constants? Or not for `.keys()`?

Reply:
https://docs.bazel.build/versions/main/skylark/lib/dict.html
> Dictionaries are iterable; iteration yields the sequence of keys in insertion order.
so yeah, I'm pretty confident this gives the last key.

Reply:
Oh, I was looking at go maps 🤦